ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```
@@ -262,6 +262,125 @@ keys:
                 type: str
                 required: true
                 description: Regular Expression
+  ip_igmp_snooping:
+    type: dict
+    display_name: IP IGMP Snooping
+    keys:
+      globally_enabled:
+        type: bool
+        default: true
+      robustness_variable:
+        type: int
+        convert_types:
+        - str
+      restart_query_interval:
+        type: int
+        convert_types:
+        - str
+      interface_restart_query:
+        type: int
+        convert_types:
+        - str
+      fast_leave:
+        type: bool
+      querier:
+        type: dict
+        keys:
+          enabled:
+            type: bool
+          address:
+            type: str
+            display_name: IP Address
+          query_interval:
+            type: int
+            convert_types:
+            - str
+          max_response_time:
+            type: int
+            convert_types:
+            - str
+          last_member_query_interval:
+            type: int
+            convert_types:
+            - str
+          last_member_query_count:
+            type: int
+            convert_types:
+            - str
+          startup_query_interval:
+            type: int
+            convert_types:
+            - str
+          startup_query_count:
+            type: int
+            convert_types:
+            - str
+          version:
+            type: int
+            convert_types:
+            - str
+      proxy:
+        type: bool
+      vlans:
+        type: list
+        primary_key: id
+        convert_types:
+        - dict
+        items:
+          type: dict
+          keys:
+            id:
+              type: str
```

vlan id should be of type int
```suggestion
              type: int
```